components/ActionSheet/tests/unit/ActionSheetTest.swift

```diff
@@ -76,4 +76,20 @@ class ActionSheetTest: XCTestCase {
     }
     
   }
+
+  func testDefaultBackgroundColor() {
+    // Then
+    XCTAssertEqual(actionSheet.backgroundColor, .white)
+  }
+
+  func testSetBackgroundColor() {
+    // Given
+    let newBackgroundColor: UIColor = .green
+
+    // When
+    actionSheet.backgroundColor = newBackgroundColor
+
+    // Then
+    XCTAssertEqual(actionSheet.backgroundColor, newBackgroundColor)
+  }
```

Comment:
You may also want to add a test to verify that `view.backgroundColor` matches `.backgroundColor` after the view is loaded:

```swift
// Given
actionSheet.backgroundColor = .green

// When
let _ = actionSheet.view // Load the view

// Then
XCTAssertEqualObjects(actionSheet.view.backgroundColor, actionSheet.backgroundColor)
```

Reply:
Done